docs/index.md

```diff
@@ -1020,45 +941,96 @@ The relevant section of the Jinja document is called  [Expressions](https://jinj
 
 In theory you could use Jinja keywords like `${% if` (as opposed to `{% if`) but it isn’t clear under what circumstances that would be necessary.
 
-## Template File Options
+### Formula functions and variables
 
-Hard-coding the exact number of records to create into a template file is not always the ideal thing.
+The functions below are designed to be used inside of formulas:
 
-You can pass options (numbers, strings, booleans) to your generator recipe from a command line.
+#### `child_index`
 
-The first step is to declare the options in your template file:
+The `child_index` variable returns a counter of how many objects from this template were generated
+during the execution of the nearest parent template. It resets each time the parent template is
+executed again.
 
 ```yaml
-- option: num_accounts
-  default: 10
+child_index: Child number ${{child_index}}
 ```
 
-If you do not specify a default, the option is required and the template will not be processed without it.
+#### `id`
 
-In your recipe, you use the value by referring to it in a formula:
+The `id` variable returns a unique identifier for the current Object/Row to allow you to construct unique identifiers.
 
 ```yaml
-- object: Account
-  count: ${{num_accounts}}
+fields:
+  name: ${{fake.last_name}} Household ${{id}}
 ```
 
-Of course you can do any math you want in the formula:
+#### `today`
+
+The `today` variable returns a date
+representing the current date. This date
+will not chanage during the execution of
+a single recipe.
+
+#### `fake.`
```

Comment:
Is this period spurious?

Reply:
No, just laziness. Clarified.

d26955c